frontend/src/pages/modelServing/screens/projects/kServeModal/ManageKServeModal.tsx

```diff
@@ -165,7 +165,10 @@ const ManageKServeModal: React.FC<ManageKServeModalProps> = ({
   // Inference Service Validation
   const storageCanCreate = (): boolean => {
     if (createDataInferenceService.storage.type === InferenceServiceStorageType.EXISTING_STORAGE) {
-      return createDataInferenceService.storage.dataConnection !== '';
+      return (
+        createDataInferenceService.storage.dataConnection !== '' ||
+        (dataConnections.length > 0 && createDataInferenceService.storage.path !== '')
```

Comment:
Is this line necessary now? I'm not sure I fully understand the case here. If we're using EXISTING_STORAGE we should always have a dataConnection name right?

Reply:
yeah It was part of me trying to secure all parts of button enabling, but you're right it's not necessary.  removed it.